### PR TITLE
Issue #6955 Fixed

### PR DESCRIPTION
**Pull request by ashishkumar-webkul**
## Issue Reference
#6955

### CODE REVIEW

**Review by Sanjaybhattwebkul — DISMISSED** [review]

File: packages/Webkul/Velocity/src/Resources/views/shop/customers/session/index.blade.php [hunk 1]
```diff
@@ -81,14 +81,18 @@ class="show-password"
...
 
                                 {{ __('shop::app.customer.login-form.show-password') }}
-
-                                <span class="control-error" v-if="errors.has('password')" v-text="errors.first('password')"></span>
-
                                 <a href="{{ route('shop.customer.forgot_password.create') }}" class=" show-password float-right">
                                     {{ __('shop::app.customer.login-form.forgot_pass') }}  
                                 </a>
 
+                                <!-- <span class="control-error" v-if="errors.has('password')" v-text="errors.first('password')"></span> -->
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Resources/views/shop/customers/session/index.blade.php:88]: Remove commented code

**Comment by Sanjaybhattwebkul** [comment]
@ashishkumar-webkul 
Now, this issue is **fixed**, please find the **screenshot** [Here](https://prnt.sc/K2dNoYhInIeJ)

File: packages/Webkul/Velocity/src/Resources/views/shop/customers/session/index.blade.php [hunk 2]
```diff
@@ -81,14 +81,18 @@ class="show-password"
...
+
+                                <!---- <a href="{{ route('shop.customer.forgot_password.create') }}" class=" show-password float-right">
+                                    {{ __('shop::app.customer.login-form.forgot_pass') }}  
+                                </a>-->
+
                                 <div class="mt10">
+                                <span class="control-error" v-if="errors.has('password')" v-text="errors.first('password')"></span>
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Resources/views/shop/customers/session/index.blade.php:95]: use tab in child tag

**Comment by ashishkumar-webkul** [comment]
@Sanjaybhattwebkul 
New pr created for the same issue: https://github.com/bagisto/bagisto/pull/6963
So closing this pr

**Pull request closed without merging**